### ISSUE TITLE
Install Rust language via wlinux-setup

### ISSUE THREAD

**Issue by cspotcode** [WhitewaterFoundry/WLinux#249]
**Describe the solution you'd like**

wlinux-setup has option to install [Rust](https://www.rust-lang.org/en-US/index.html) toolchain

**Describe alternatives you've considered**

The rust website offers this script to install rust:

```
curl https://sh.rustup.rs -sSf | sh
```

...but it would be nice if WLinux had this as a built-in option.

**Comment by grufwub** [comment]
Working on that here: https://github.com/grufwub/WLinux/tree/rust-lang

Still need to give this a test, and work out what flags are best supplied to the rustup installer (platform, toolchain version etc).

**Comment by grufwub** [comment]
See #258

**Comment by grufwub** [comment]
Merged into WLinux/development, now waiting on #256
